### ISSUE TITLE
Edit content field and remove type field from 404.yml credits file

### ISSUE THREAD

**Issue by SAUMILDHANKAR** [hackforla/website#2633]
### Dependency
- [x] - #2642

### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, I want to edit the content field and remove the type field from a credit's yml file so that redundant code is removed and the code is easier to understand.

### Details
Currently, in each credit's yml file, there is a `content` field and a `type` field, which contains similar information. Since the fields are redundant, we decided to remove the `type` field.  

Also, we changed the `content` field to the `content-type` field to make it clearer what its purpose is. The possible values for `content-type` field are image, video, or audio; other types of content can be added if necessary. In the future, this will allow developers a way to differentiate easily between different types of content in order to show each credit's media file correctly on the website.

### Action Items
For the file `_data/internal/credits/404.yml`, do the following:

- [x] Change line 4 for the content field from
  ```
  content: Image
  ```
  to
  ```
  content-type: image
  ```
- [x] Remove line 11 for the type field
   ```
   type: icon
   ```
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #2775 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please move issue #2775 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is tracked in the epic #2775.

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_data/internal/credits/404.yml`

- Click on the links below to see the source file code:
https://github.com/hackforla/website/blob/6bb9e8aa9aff0d44a46419ec54ebc342ae073bf0/_data/internal/credits/404.yml#L4
https://github.com/hackforla/website/blob/6bb9e8aa9aff0d44a46419ec54ebc342ae073bf0/_data/internal/credits/404.yml#L11

Availability for this week: 6 hrs
My estimated ETA for completing this issue: 10 minutes

**Comment by pattshreds** [comment]
Availability for this week: Today 1.17.2022 all day.
My estimated ETA for completing this issue: Within the next hour.

**Comment by scorbz9** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
Answer: ~4.5 hours
